kotlin/workflow-ui/core-android/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2017 Square Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<resources>
+  <!-- View Tag that holds the function to update a view created via ViewRegistry. -->
+  <item type="id" name="view_show_rendering_function"/>
+  <!-- View Tag used for back button handlers. -->
+  <item type="id" name="view_back_handler"/>
```

Comment:
Nit: dunno if you are strict about alphabetical order of values in resources, fine by me.

Reply:
This file was just moved, leaving as-is.